### DISCUSSION TITLE
Contribution to WC_REST_Customers_V1_Controller

### DISCUSSION THREAD

**Discussion by ReyJust** [woocommerce/woocommerce#51809] [General]
Hello,
I would like to know if it's possible to contribute to `WC_REST_Customers_V1_Controller` class? I am in doubt because I can see "versioning" in extended classes `WC_REST_Customers_V2_Controller` or `WC_REST_Customers_Controller`, maybe it can't be modified for backward compatibility problem or something else? No idea 😅 

Thanks ! 😆

**Comment by coreymckrill** [comment]
Hi, the `WC_REST_Customers_Controller` is actually the current version (v3). So any changes you made in the v1 controller would be inherited by the other two versions, unless they are in methods that are explicitly overridden.

What sort of changes are you looking to make? It would probably be best to start with an [enhancement request issue](https://github.com/woocommerce/woocommerce/issues/new?assignees=&labels=type%3A+enhancement&projects=&template=2-enhancement.yml&title=%5BEnhancement%5D%3A+).
